src/main/java/Duke.java

```diff
@@ -1,10 +1,183 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static int pendingTaskCount = 0;
+
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String HORIZONTAL_LINE =
+            "____________________________________________________________";
+
+    private static final int REQUIRED_NUMBER_OF_ARGUMENT_FOR_TODO = 2;
+    private static final int REQUIRED_NUMBER_OF_ARGUMENT_FOR_EVENT = 4;
+    private static final int REQUIRED_NUMBER_OF_ARGUMENT_FOR_DEADLINE = 4;
```

Comment:
I don't quite understand the purposes of these variables. May I know what are they used for? Also, I feel that the name is a bit lengthy, and the grammar is a bit off; for example, it should be REQUIRED_NUMBER_OF_ARGUMENT**S**_FOR_TODO.

Reply:
They are used for checking if the user has supplied the correct number of arguments for the commands. I will try to think of a better name to replace "REQUIRED_NUMBER_ARGUMENTS_FOR_TODO"! As for the grammar issue, I guess it's pretty subjective, both ways should be fine regardless.